source/summary.rst

```diff
@@ -0,0 +1,16 @@
+Summary
+=======
+
+.. toctree::
+   :maxdepth: 1
+
+   index
+   key-concepts/index
+   coding-the-legislation/index
+   openfisca-web-api/index
```

Comment:
It would make sense add them consecutively (API and Python)? Just easier to read.

Reply:
I'll postpone this do another PR about doc content :)